ui/src/routes/devices.$id.settings.keyboard.tsx

```diff
@@ -51,7 +51,6 @@ export default function SettingsKeyboardRoute() {
       />
 
       <div className="space-y-4">
-        { /* this menu item could be renamed to plain "Keyboard layout" in the future, when also the virtual keyboard layout mappings are being implemented */ }
```

Comment:
Is there a reason to remove this? I felt it served a purpose, as a reminder that `title="Paste text"` can be renamed to `title="Keyboard layout"` once virtual keyboards have caught up with their keyboard layouts.

`title="Paste text"` only exists as an intermediary solution, since there are different development speeds for "paste text" and "virtual keyboard" functionality. This should be cleaned up later. Maybe there is a better place to persist this information?

Reply:
I've come around to thinking it is clearer the way it is.. the setting is and should be related to the the remote's OS keyboard layout or things just won't work.

Reply:
... btw, working on the Virtual Keyboard again... getting the Ctrl/Alt to track "down" states like it does for Shift so it's more like a touch keypad... which will also allow sending things like Ctrl+Ctrl+1 from the Virtual Keyboard as well as the physical keyboard.

Reply:
Exactly! It would be confusing for users to have a keyboard layout setting for "paste box" and another keyboard layout setting for "virtual keyboard", that should be the same eventually, when development on layouts for "virtual keyboards" has caught up. Hence the reminder above.

Reply:
I'll restore the comment until such time as it needs to change to something more valuable